airflow/executors/celery_executor.py

```diff
@@ -59,17 +58,17 @@
 
 
 @app.task
-def execute_command(command_to_exec: str) -> None:
+def execute_command(command_to_exec: List[str]) -> None:
     """Executes command."""
     log = LoggingMixin().log
     log.info("Executing command in Celery: %s", command_to_exec)
-    env = os.environ.copy()
     try:
-        subprocess.check_call(command_to_exec, stderr=subprocess.STDOUT,
-                              close_fds=True, env=env)
-    except subprocess.CalledProcessError as e:
-        log.exception('execute_command encountered a CalledProcessError')
-        log.error(e.output)
+        parser = CLIFactory.get_parser()
+        # drop "airflow"
+        command_to_exec = command_to_exec[1:]
+        args = parser.parse_args(command_to_exec)
+        args.func(args)
```

Comment:
Maybe we can put this logic into a function? It would help in future refactor.

Reply:
I wanted to delete CLI references completely during another refactoring, but it makes sense to create functions now.

Reply:
Having a function will make it easier imho ✅

Reply:
In my opinion, this will not affect it in any way. The code I am writing now will delete after refactoring. Now, when I create new functions I will also have to write tests, so this code will be more than 3 lines.

Reply:
If we use it only here then value of such function is poor, but I think we can re-use it in other places. Then  further refactoring of executing task will be easier because only this function will have to be refactored.